### PR TITLE
Update and rename what-is-a-comment?.md

### PR DESCRIPTION
**Pull request by sammillersmith**

### CODE REVIEW

File: content/language-agnostic/encyclopedia/what-is-a-comment.md [hunk 1]
```diff
@@ -6,27 +6,28 @@ Subjects:
...
+Some programming languages support only one type of comment. For example, Python comments are single-line comments: They start with `#` and continue to the end of the line. Other languages employ both single-line and multi-line comments. For example, C and C++ have single-line comments that start with `//` and multi-line comments between `/*` and `*/` that can span multiple lines.
 
 ## Uses
 
 How best to make use of comments is subject to dispute. Here are some common ways comments are used:
 
 ### Code Description
 
-Comments can be used to summarize code or to explain the programmer's intent. According to this school of thought, restating the code in plain English is considered superfluous; the need to re-explain code may be a sign that it is too complex and should be rewritten, or that the naming is bad.
+Comments can be used to summarize code or to explain the programmer's intent. According to this school of thought, restating the code in plain English is superfluous; the need to re-explain code may be a sign that it is too complex and should be rewritten, or that the naming is bad.
```
Comment (sammillersmith) [inline content/language-agnostic/encyclopedia/what-is-a-comment.md:30]: Should there be a preceding subsection explaining the school of thought that comments should, in fact, restate the code in plain English? As written, the difference between restating the code in plain English, which this camp apparently thinks is bad, and summarizing the code, which this camp apparently thinks is good, is unclear.

File: content/language-agnostic/encyclopedia/what-is-a-comment.md [hunk 2]
```diff
@@ -6,27 +6,28 @@ Subjects:
...
 
 > "Good comments don't repeat the code or explain it. They clarify its intent. Comments should explain, at a higher level of abstraction than the code, what you're trying to do." (Steve McConnell)
```
Comment (sammillersmith) [inline content/language-agnostic/encyclopedia/what-is-a-comment.md:32]: Readers may not know who Steve McConnell is. Who Steve McConnell is, as well as the source of this quote, should likely be cited.

File: content/language-agnostic/encyclopedia/what-is-a-comment.md [hunk 3]
```diff
@@ -6,27 +6,28 @@ Subjects:
...
```
Comment (sammillersmith) [inline content/language-agnostic/encyclopedia/what-is-a-comment.md:33]: I can't comment on line 35 (per the original file) because I didn't edit it, but I wanted to flag this thought that I wrote down when I read it: "How so? Could use some more details." Essentially, it isn't clear exactly _how_ comments can be used for debugging.

File: content/language-agnostic/encyclopedia/what-is-a-comment.md (hunk 1 above)
Comment (sonnynomnom) [reply content/language-agnostic/encyclopedia/what-is-a-comment.md:30]: sam, what did u study in college? did u study english??

Comment (sammillersmith) [reply content/language-agnostic/encyclopedia/what-is-a-comment.md:30]: Philosophy, which was somehow even less practical

**Review by sonnynomnom — APPROVED** [review]

File: content/language-agnostic/encyclopedia/what-is-a-comment.md (hunk 2 above)
Comment (sonnynomnom) [reply content/language-agnostic/encyclopedia/what-is-a-comment.md]: deleted!

File: content/language-agnostic/encyclopedia/what-is-a-comment.md (hunk 1 above)
Comment (sonnynomnom) [reply content/language-agnostic/encyclopedia/what-is-a-comment.md]: changed this paragraph a bit.

File: content/language-agnostic/encyclopedia/what-is-a-comment.md (hunk 3 above)
Comment (sonnynomnom) [reply content/language-agnostic/encyclopedia/what-is-a-comment.md:31]: edited line 35 (added "through process of elimination")

**Comment by sonnynomnom** [comment]
@sammillersmith i made some fixes. feel free to go over again!

**Comment by sammillersmith** [comment]
Looks excellent to me. Merging.

**Review by sonnynomnom — APPROVED** [review]
